$OpenBSD$

While being deprecated and undocumented, SANE_CAP_ALWAYS_SETTABLE is
still used by (at least) xsane.

--- include/sane/sane.h.orig	Mon May 11 17:17:41 2009
+++ include/sane/sane.h	Mon May 11 17:17:57 2009
@@ -115,6 +115,7 @@ SANE_Device;
 #define SANE_CAP_AUTOMATIC		(1 << 4)
 #define SANE_CAP_INACTIVE		(1 << 5)
 #define SANE_CAP_ADVANCED		(1 << 6)
+#define SANE_CAP_ALWAYS_SETTABLE	(1 << 7)
 
 #define SANE_OPTION_IS_ACTIVE(cap)	(((cap) & SANE_CAP_INACTIVE) == 0)
 #define SANE_OPTION_IS_SETTABLE(cap)	(((cap) & SANE_CAP_SOFT_SELECT) != 0)
